### PR TITLE
Users list roles

### PR DESCRIPTION
**Pull request by KateVuilova**

### CODE REVIEW

File: src/containers/UsersPage/UsersPage.js [hunk 1]
```diff
@@ -91,10 +50,67 @@ class UsersPage extends Component {
...
 
+    handleChangeRole(userId, accountId, isAdmin) {
+        this.props.changeUserRole(userId, accountId, isAdmin);
+    }
+
     render() {
         if (this.props.usersListLoading || this.props.currentAccountLoading) {
             return <div>Loading...</div>;
         }
+        let columnNamesInitial = [
```
Comment (mrmckeb) [inline src/containers/UsersPage/UsersPage.js]: These should be declared at the top of the file, as here they're being 'recreated' on every render. It looks like they were moved - why was that?

File: src/components/List/List.js [hunk 2]
```diff
@@ -44,16 +44,31 @@ class List extends Component {
...
                 } else {
-                    return (
-                        <td key={index} className={column.className || ""}>
-                            {value[column.name]}
-                        </td>
-                    );
+                    if (column.name === "isAdmin") {
+                        return (
+                            <td key={index} className={column.className || ""}>
+                                {value[column.name] === true ? "Admin" : "User"}
```
Comment (mrmckeb) [inline src/components/List/List.js]: The only difference between these seems to be the content. Could this code change the content of the `td` without returning the `td` itself?

File: src/containers/UsersPage/UsersPage.js (hunk 1 above)
Comment (mrmckeb) [reply src/containers/UsersPage/UsersPage.js]: I understand the below were moved because they now have logic.

Comment (KateVuilova) [reply src/containers/UsersPage/UsersPage.js]: Done

File: src/components/List/List.js (hunk 2 above)
Comment (KateVuilova) [reply src/components/List/List.js]: I need changes from other PR and then I will fix it.

**Pull request merged**